src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -99,4 +109,17 @@ public ResponseEntity deleteFromFollowing(@PathVariable("userId") Long user, @Pa
     userService.deleteFromFollowing(user, newUser);
     return ResponseEntity.status(200).build();
   }
+
+  @PostMapping("/user/{email}")
+  public ResponseEntity logInUser(@PathVariable("email") String email, @RequestBody String password) {
+    User user = userService.logIn(email, password);
+    if (user == null) {
+      return ResponseEntity.status(204).body("Wrong email ");
+    } else if (!user.getPassword().equals(password)) {
+      return ResponseEntity.status(205).body("Incorrect passwoord");
+    } else {
+      userEndPoint = user;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make the enclosing method "static" or remove this set. ([squid:S2696](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS2696))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=134&issues=AWVbiOHjZiUEgfxFxpi6&open=AWVbiOHjZiUEgfxFxpi6)

[ISSUE_KEY]: AWVbiOHjZiUEgfxFxpi6